lua/entities/gmod_wire_expression2/core/custom/acffunctions.lua

```diff
@@ -1000,6 +1000,16 @@ e2function number entity:acfDragCoef()
 	return Round(this.BulletData.DragCoef / ACF.DragDiv, 10)
 end
 
+-- Returns the fin multiplier of the missile/bomb
+e2function number entity:acfFinMul()
+	if not IsACFEntity(this) then return 0 end
+	if RestrictInfo(self, this) then return 0 end
+	if not this.BulletData then return 0 end
+	if not this.BulletData.Id then return 0 end
+
+	return ACF.Weapons.Guns[this.BulletData.Id].round.finmul
```

Comment:
I'm not sure if I could do this better, but this is how I saw it done in `acf3missiles/lua/entities/acf_missile/init.lua:99`

Reply:
Could add some extra checks to make sure the gun data exists and contains a round table inside
```lua
local GunData = ACF.Weapons.Guns[this.BulletData.Id]

if not GunData then return 0 end
if not GunData.round then return 0 end

return GunData.round.finmul or 0
```